script/build

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
script/compile just compiles the dll and repacks it to include dependenices.  script/build does that, and also wraps it up for release to nuget.

I was 51% in favor of doing it this way, and 49% in favor making the current compile be called build, and then writing a script/package to take care of nuget.  I came down on this side because if someone does what they learned in our other systems and does script/build, they will get the most possible amount of building things.